bigchaindb_driver/driver.py

```diff
@@ -39,6 +40,12 @@ def __init__(self, *, public_key, private_key, node):
         self.public_key = public_key
         self.private_key = private_key
         self.node = node
+        self.transport = transport_class(node)
+
+    def retrieve(self, txid):
+        path = '/transactions' + '/' + txid
```

Comment:
Yes. I thought of handling this  once we have decided on how to handle trailing slashes (bigchaindb/bigchaindb#585)